samples/ApptentiveSample/MyApplication.cs

```diff
@@ -17,10 +17,12 @@ public override void OnCreate()
         {
             base.OnCreate();
 
-            var configuration = new ApptentiveConfiguration("Your Apptentive Key", "Your Apptentive Signature");
-            configuration.SetLogLevel(ApptentiveLog.Level.Verbose);
-            configuration.SetShouldSanitizeLogMessages(false);
-            Apptentive.Register(this, configuration);
+            var configuration = new ApptentiveKit.Android.ApptentiveConfiguration("ANDROID-XAMARIN-ANDROID", "22527498ae05c9605dd746cd7c5cb33c");
```

Comment:
As this will be public repo, let's not add the key and signature.

Reply:
Also, this has already been committed in this PR, which is public... Should we handle this in any way @frankus ?

Reply:
@frankus ?

Reply:
Not gonna worry about this